.github/workflows/windows.yml

```
@@ -81,6 +81,6 @@ jobs:
             ${{ vars.GSTREAMER_S3_UPLOAD_BUCKET && vars.GSTREAMER_S3_UPLOAD_PREFIX && format('s3://{0}/{1}', vars.GSTREAMER_S3_UPLOAD_BUCKET, vars.GSTREAMER_S3_UPLOAD_PREFIX) || '' }}
 
       - name: Annotate workflow run with GStreamer version
-        if: steps.build-gstreamer.outputs.gstreamer-version
+        if: steps.setupo-gstreamer.outputs.gstreamer-version
```

Typo, which definitely made me chuckle a bit
```suggestion
        if: steps.setup-gstreamer.outputs.gstreamer-version
```

---

I'm amazed nothing caught that as an error/warning given the variety of other things I've read.

Fixed in 636a6dea.